x-pack/legacy/plugins/code/server/routes/search.ts

```diff
@@ -261,3 +287,15 @@ export function commitSearchRoute(router: CodeServerRouter, log: Logger) {
     },
   });
 }
+
+async function getScope(req: RequestFacade, repoScope: string | string[]): Promise<string[]> {
+  let scope: string[] = await getReferenceHelper(req.getSavedObjectsClient()).findReferences();
```

Comment:
Since we just bring the `SavedObjectsClient` back and the platform team is migrating to new platform, we might consider directly use the New Platform's API right away. Let me double check with them if the SavedObjectsClient is ready there.

Reply:
they are still a couple of weeks to go. So I just add an item to the NP migration issue we have.https://github.com/elastic/code/issues/1596